nucliadb_protos/writer.proto

```diff
@@ -151,12 +151,46 @@ message GetLabelsRequest {
     knowledgebox.KnowledgeBoxID kb = 1;
 }
 
+message NewEntitiesGroupRequest {
+    knowledgebox.KnowledgeBoxID kb = 1;
+    string group = 2;
+    knowledgebox.EntitiesGroup entities = 3;
+}
+
+message NewEntitiesGroupResponse {
+    enum Status {
+        OK = 0;
+        ERROR = 1;
+        KB_NOT_FOUND = 2;
+        ALREADY_EXISTS = 3;
+    }
+    Status status = 1;
+}
+
+
 message SetEntitiesRequest {
     knowledgebox.KnowledgeBoxID kb = 1;
     string group = 2;
     knowledgebox.EntitiesGroup entities = 3;
 }
 
+message UpdateEntitiesGroupRequest {
+    knowledgebox.KnowledgeBoxID kb = 1;
+    string group = 2;
+    map<string, knowledgebox.Entity> add = 3;
+    map<string, knowledgebox.Entity> update = 4;
```

Comment:
Can you add a comment indicating what is the expected value of the map's key?

Reply:
They are the entity ids. Same as with EntitiesGroup entities map. I'll add a comment though